src/components/auth/login/WelcomeScreen.js

```diff
@@ -42,14 +42,14 @@ const WelcomeScreen = ({ styles, screenProps, navigation }) => {
     fireEvent(SIGNUP_SELECTED)
 
     const options = { screen: 'signup' }
-    if (isMobileNative) options.regMethod = REGISTRATION_METHOD_SELF_CUSTODY
+    if (isIOSNative) options.regMethod = REGISTRATION_METHOD_SELF_CUSTODY
```

Comment:
How about IOS ? When Torus will provide its support in RN package ? I checked - Swift SDK it's ready so they just need to write a simple RCTBridgeModule wrapping it

@sirpy Please advice - could we write our Torus for IOS implementation if they won't provide it near time (e.g. the next week ? Of course our implementation will be very raw and will implement just triggerLogin() method. We'll use it just temporarily unit Torus provides it own solution. On our side it will be just an package patch adding .swift file with implementation and RCT_EXPORT_METHOD macro to the RNTorusDirectSdk.m

Reply:
We'll wait for the Torus team implementation may be released this Friday